### PR TITLE
MySQL: Update documentation around timezone

### LINKED ISSUE #75825: MySQL: Explain the time zone option more
**What is the chore?**:

Clear out the documentation for mysql data source around the time zone field.

### PR DESCRIPTION
**Pull request by zoltanbedi**
This PR adds a key sentence to the mysql timezone option to make it clear what to do when the database is set to somehting else other than UTC.

**Which issue(s) does this PR fix?**:

Fixes #75825

### CODE REVIEW

**Review by gabor — APPROVED** [review]
i think this is a good change.

but in the future i think we could improve it in two directions:
1. maybe literally start the description with something like "you nearly always want to leave this at 00:00" ...
2. (this may be hard).. somehow rewrite the whole text and explain to the user how to set the number based on different scenarios.

File: docs/sources/datasources/mysql/_index.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Administrators can also [configure the data source via YAML](#provision-the-data
...
+| **Session Timezone**          | Specifies the time zone used in the database session, such as `Europe/Berlin` or `+02:00`. Required if the timezone of the database (or the host of the database) is set to something other than UTC. Set this to `+00:00` so grafana can handle times properly. Set the value used in the session with `SET time_zone='...'`. If you leave this field empty, then the time zone is not updated. For more information, refer to the [MySQL documentation](https://dev.mysql.com/doc/en/time-zone-support.html).                                                                                                                                           |
```
Comment (lwandz13) [inline docs/sources/datasources/mysql/_index.md:51]:
```suggestion
| **Session Timezone**          | Specifies the timezone used in the database session, such as `Europe/Berlin` or `+02:00`. Required if the timezone of the database (or the host of the database) is set to something other than UTC. Set this to `+00:00` so Grafana can handle times properly. Set the value used in the session with `SET time_zone='...'`. If you leave this field empty, the timezone will not be updated. For more information, see [ MySQL Server Time Zone Support](https://dev.mysql.com/doc/en/time-zone-support.html).                                                                                                                                           |
```

**Review by lwandz13 — DISMISSED** [review]
Minor nits

File: docs/sources/datasources/mysql/_index.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Administrators can also [configure the data source via YAML](#provision-the-data
...
+| **Session Timezone**          | Specifies the timezone used in the database session, such as `Europe/Berlin` or `+02:00`. Required if the timezone of the database (or the host of the database) is set to something other than UTC. Set this to `+00:00` so Grafana can handle times properly. Set the value used in the session with `SET time_zone='...'`. If you leave this field empty, the timezone will not be updated. For more information, see [ MySQL Server Time Zone Support](https://dev.mysql.com/doc/en/time-zone-support.html).                                                                                                                                          |
```
Comment (jdbaldry) [inline docs/sources/datasources/mysql/_index.md:51]: I know this partially reverts a previous suggestion but we should prefer "refer to" over "see" (https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#links-and-references).

I also imagine it is going to break `prettier` so feel free to incorporate however you choose.

```suggestion
| **Session Timezone**          | Specifies the timezone used in the database session, such as `Europe/Berlin` or `+02:00`. Required if the timezone of the database (or the host of the database) is set to something other than UTC. Set this to `+00:00` so Grafana can handle times properly. Set the value used in the session with `SET time_zone='...'`. If you leave this field empty, the timezone will not be updated. For more information, refer to [MySQL Server Time Zone Support](https://dev.mysql.com/doc/en/time-zone-support.html).                                                                                                                                          |
```

**Review by jdbaldry — APPROVED** [review]
Approving to unblock but please consider incorporating the suggestion.

**Pull request merged**